presto-tests/src/main/java/com/facebook/presto/tests/statistics/StatisticsAssertion.java

```diff
@@ -99,6 +99,15 @@ public Checks verifyCharacterColumnStatistics(String columnName, MetricCompariso
             return this;
         }
 
+        public Checks verifyNoColumnStatistics(String columnName)
+        {
+            noEstimate(nullsFraction(columnName));
```

Comment:
Some of the methods in this class start with `verify` some of them don't.  This bit is confusing. Maybe `expectNoEstimate`? or `verifyNoEstimate`? Although it should rather be a part of a separate PR.

Reply:
Hm... I somewhat agree. However, let's put this cleanup aside for now, since this PR doesn't intro any new mess here.